src/Microsoft.Azure.WebJobs.Host/Executors/FunctionExecutor.cs

```diff
@@ -227,16 +227,9 @@ internal static async Task HandleExceptionAsync(TimeoutAttribute timeout, Except
         private async Task<string> ExecuteWithLoggingAsync(IFunctionInstanceEx instance, FunctionStartedMessage message,
             FunctionInstanceLogEntry instanceLogEntry, ParameterHelper parameterHelper, ILogger logger, CancellationToken cancellationToken)
         {
-            IFunctionOutputDefinition outputDefinition = null;
-            IFunctionOutput outputLog = null;
-            ITaskSeriesTimer updateOutputLogTimer = null;
-            TextWriter functionOutputTextWriter = null;
-
-            IFunctionOutputLogger outputLogger = _functionOutputLogger;
-            outputDefinition = await outputLogger.CreateAsync(instance, cancellationToken);
-            outputLog = outputDefinition.CreateOutput();
-            functionOutputTextWriter = outputLog.Output;
-            updateOutputLogTimer = StartOutputTimer(outputLog.UpdateCommand, _exceptionHandler);
+            var outputDefinition = await _functionOutputLogger.CreateAsync(instance, cancellationToken);
+            var outputLog = outputDefinition.CreateOutput();
+            var updateOutputLogTimer = StartOutputTimer(outputLog.UpdateCommand, _exceptionHandler);
```

Comment:
There's a lot of changes to `var` throughout the PR -- is that an auto-formatting thing you run or was there a perf reason for it?

Reply:
Just code readability. As part of this work I was examining the entire pipeline and making readability tweaks as I went.